### DISCUSSION TITLE
GSoC 2023 Proposal Template or Previous Proposals

### DISCUSSION THREAD

**Discussion by Soumya624** [GeomScale/volesti#274] [Q&A] [answered]
Hi,

I was going through the Gitter channel of GeomScale and couldn't find any previous year's GSoC proposal for reference. Found GSoD though. So, if anyone can help me with this or the proposal template (if any), that would be great help.

Thnaks!

**Comment by vissarion** [comment, marked as answer]
Please see GeomScale's [wiki page for GSoC 2023](
https://github.com/GeomScale/gsoc23/wiki)
